core/assets/bundles/bundle.properties

```diff
@@ -1082,6 +1082,7 @@ block.wave.name = Wave
 block.swarmer.name = Swarmer
 block.salvo.name = Salvo
 block.ripple.name = Ripple
+block.clapper.name = Clapper
```

Comment:
kind of an odd name

Reply:
initially the turret was called a ```plastanium cracker```, then I decided to make the name more fluoristic and called ```xenolit```, but in the end I returned to the old name and decided to leave it just a ```clapper```